target_chains/ethereum/sdk/solidity/PythUtils.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+library PythUtils {
+    /// @notice Converts a Pyth price to a uint256 with a target number of decimals
+    /// @param price The Pyth price
+    /// @param expo The Pyth price exponent
+    /// @param targetDecimals The target number of decimals
+    /// @return The price as a uint256
+    /// @dev Function will loose precision if targetDecimals is less than the Pyth price decimals
```

Comment:
"lose" not "loose"

Reply:
may be worth explicitly noting that the method truncates any digits that cannot be represented in the target number of decimals (which is why you get 0 in the case below)